src/main/java/com/semmle/jira/addon/util/JiraUtils.java

```diff
@@ -193,7 +193,8 @@ public static void createConfigKeyCustomField() throws GenericEntityException {
 
       builder.setManaged(true);
       builder.setConfigurationItemAccessLevel(ConfigurationItemAccessLevel.LOCKED);
-      builder.setSource(Constants.PLUGIN_KEY);
+      builder.setSource(
+          ManagedConfigurationItemService.SOURCE_PREFIX_PLUGIN + Constants.PLUGIN_KEY);
```

Comment:
Needs the same prefix added in the check on line 170.

Reply:
Ah thanks for spotting that!

Reply:
I amended the commit, it should be better now.